android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -755,6 +766,15 @@ private DrmSessionManager initializePlayerDrm(ReactExoplayerView self) {
                 eventEmitter.error(getResources().getString(errorStringId), e, "3003");
                 return null;
             }
+        } else if (extension.equals(DOWNLOADED_CONTENT)) {
+            try {
+                drmSessionManager = buildDrmSessionManager(self.assetId);
+            } catch (Exception ex) {
+                self.playerNeedsSource = true;
+                Log.e("ExoPlayer Exception", "Failed to initialize Player!");
+                Log.e("ExoPlayer Exception", ex.toString());
```

Comment:
as mentioned in slack, rather than passing `ex.toString()` we should always just pass ex to the logger.

Exception.toString is generally useless by itself, eg `new NullPointerException().toString()` will give you the word "null", which tells you nothing other than the error logging isn't good : )

Reply:
updated, thanks